### PR TITLE
Feature/ATG-58 Figures Matrix

### PR DESCRIPTION
**Pull request by IraCherginyshka**
Create matrix for all figures

### CODE REVIEW

File: src/app/constants/figures-matrix.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const FiguresMatrix = [
```
Comment (oleksandrstarov) [inline src/app/constants/figures-matrix.ts]: No need to use export default - we can use export const FiguresMatrix

Comment (oleksandrstarov) [inline src/app/constants/figures-matrix.ts]: Please rename file to figures-matrix.const.ts

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

File: src/app/constants/figures-matrix.ts (hunk 1 above)
Comment (IraCherginyshka) [reply src/app/constants/figures-matrix.ts]: Done

Comment (IraCherginyshka) [reply src/app/constants/figures-matrix.ts]: Done

**Pull request merged**